src/Gui/GuiApplication.cpp

```diff
@@ -75,6 +75,16 @@ bool GUIApplication::notify (QObject * receiver, QEvent * event)
             (int)event->type());
         return false;
     }
+
+    // https://github.com/FreeCAD/FreeCAD/issues/16905
+    std::string exceptionWarning =
+#if FC_DEBUG
+        "Exceptions must be catched before they go through Qt."
```

Comment:
Sorry for the small nitpick but wondering if it's preferable to have correct english grammar or cpp grammar, as in catched->caught

Reply:
Thanks, I've fixed it now.